### DISCUSSION TITLE
SriLanka users cannot register with Vercel

### DISCUSSION THREAD

**Discussion by Kajanan02** [vercel/community#3363] [Help]
### Summary

Summary
Hello,
I live in Sri Lanka. To register with Vercel, I need to verify my phone number. But SriLanka does not support telephone numbers. How can I solve this issue?

### Steps to Reproduce

Help

**Comment by amyegan** [comment]
Sorry you're having trouble registering, @Kajanan02. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
